rules/application/antivirus/av_ransomware.yml

```
@@ -6,7 +6,7 @@ references:
     - https://www.nextron-systems.com/?s=antivirus
 author: Florian Roth
 date: 2022/05/12
-modified: 2021/12/02
+modified: 2022/05/12
```

The last change was made in this PR https://github.com/SigmaHQ/sigma/pull/3625
```suggestion
modified: 2022/10/24
```